src/SourceBuild/content/prep.sh

```diff
@@ -6,13 +6,18 @@
 ###   installing the version of dotnet referenced in global.json
 ###
 ### Options:
-###   --no-artifacts              Exclude the download of the previously source-built artifacts archive
-###   --no-bootstrap              Don't replace portable packages in the download source-built artifacts
-###   --no-prebuilts              Exclude the download of the prebuilts archive
-###   --no-sdk                    Exclude the download of the .NET SDK
-###   --runtime-source-feed       URL of a remote server or a local directory, from which SDKs and
-###                               runtimes can be downloaded
-###   --runtime-source-feed-key   Key for accessing the above server, if necessary
+###   --no-artifacts                Exclude the download of the previously source-built artifacts archive
+###   --no-bootstrap                Don't replace portable packages in the download source-built artifacts
+###   --no-prebuilts                Exclude the download of the prebuilts archive
+###   --no-sdk                      Exclude the download of the .NET SDK
+###   --runtime-source-feed         URL of a remote server or a local directory, from which SDKs and
+###                                 runtimes can be downloaded
+###   --runtime-source-feed-key     Key for accessing the above server, if necessary
+###   --smoke-test-prereqs-path     Directory where the smoke test prereqs packages should be downloaded to.
```

Comment:
Depending on the resolution to my previous comments, this may the the only new parameter.  If that's the case I think this script would be more dev friendly if there was a 1. a flag to enable smoke test prereq gathering 2.the path was defaulted with an env override?

Reply:
I don't know how we could have a prep script that didn't include a feed option. There has to be a way to specify the DSP feed.

Reply:
Makes sense.  I still find the smoke-test-prereqs-path option to be a bit inconsistent - there is no equivalent option to specify the artifacts path today.  Because of that, I towards this should be defaulted.  This would require the addition of a n option to indicate you want the prereqs to be gathered.